src/AddressRegistry.Projections.Wms/AddressDetailV2/AddressDetailV2Projections.cs

```diff
@@ -0,0 +1,479 @@
+namespace AddressRegistry.Projections.Wms.AddressDetailV2
+{
+    using System;
+    using Be.Vlaanderen.Basisregisters.GrAr.Legacy;
+    using Be.Vlaanderen.Basisregisters.GrAr.Legacy.Adres;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore;
+    using Be.Vlaanderen.Basisregisters.Utilities.HexByteConvertor;
+    using NetTopologySuite.Geometries;
+    using NetTopologySuite.IO;
+    using NodaTime;
+    using StreetName;
+    using StreetName.Events;
+
+    [ConnectedProjectionName("WMS adressen")] // TODO moeten we dit aanpassen naar V2 ?
```

Comment:
@ArneD what should be the ConnectedProjectionName and ~Description?

Reply:
same as V1